### DISCUSSION TITLE
More accurate clearance heights for flat rides

### DISCUSSION THREAD

**Discussion by astraylife1** [OpenRCT2/OpenRCT2#22141] [Feature requests]
The spiral slide has a clearance height of 16 units while the circus which is much smaller has the same clearance height of 16 units. But looking at the Top Spin which is definitely taller than the circus building, the clearance height is only 14 units. And then there is at the double decker carousel which is just as tall as the spiral slide but only has a clearance height of 8 units (probably because Frontier never updated it from the standard merry go round). Just seems like these should all be updated to numbers that appear more accurate.

**Comment by Gymnasiast** [comment]
Can you create a PR?

**Reply by astraylife1** [reply]
I wish I had any skills that could be of use, but unfortunately I wouldn’t even know where to begin

**Reply by J-PIE-314** [reply]
You can check out how Gymnasiast fixed the cash machine height: [https://github.com/OpenRCT2/OpenRCT2/pull/18827/files](https://github.com/OpenRCT2/OpenRCT2/pull/18827/files)

**Reply by astraylife1** [reply]
I appreciate the reference, however with zero code experience, that all reads foreign to me. If I knew how to fix this myself I would have, but alas I made a feature request as that seemed most appropriate for my abilities or lack thereof.

**Comment by J-PIE-314** [comment]
Some of the requested changes will be implemented in PR #22744.
Any more clearance changes will require separation of drawing code, or z-fighting.
